packages/commons/package.json

```diff
@@ -9,13 +9,14 @@
     "test:watch": "jest --watch",
     "test:compilation": "tsc --noEmit",
     "postinstall": "yarn build",
-    "build": "tsc -d",
+    "build": "tsc",
     "build:clean": "rm -rf build"
   },
   "private": true,
   "dependencies": {
     "@types/node-fetch": "^2.3.3",
     "@hapi/hapi": "^20.0.1",
+    "@types/hapi__hapi": "^20.0.0",
```

Comment:
@metrics service complained in its build if commons didnt have these types in production install.  No idea why that service complained.